### PR TITLE
Truncate long snvs names in general report

### LINKED ISSUE #1760: Truncate very long variant names in general report
Otherwise the layout will be ugly

Example from this case: https://scout.scilifelab.se/cust002/F0031880

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #1760 
When snv variants have very long display names the general report looks ugly:

image

**How to test**:
1. On a local instance of mongodb change display name of this variant : _id: a76ebd54e531bb276638544b187c14fd to `17_7470283_ACGGGTAAAACCATTGGTATTATATTAGGCCCGAGGAGACTTTCTCTCTCTTCGGGGAGAGGCTTCTCTCTCTTCTCTC_A_clinical`
1. Make sure to modify also variant reference to `ACGGGTAAAACCATTGGTATTATATTAGGCCCGAGGAGACTTTCTCTCTCTTCGGGGAGAGGCTTCTCTCTCTTCTCTC`
1. Modify also simple id to `17_7470283_ACGGGTAAAACCATTGGTATTATATTAGGCCCGAGGAGACTTTCTCTCTCTTCGGGGAGAGGCTTCTCTCTCTTCTCTC_A`

1. On a local installation of scout go to the same variant: http://localhost:5000/cust000/643594/a76ebd54e531bb276638544b187c14fd
 and mark it as causative, then dismiss it with many reasons, including the "inherited from unaffected" tag.
1. Check the difference in the general report using master and this branch

**Expected outcome**:
Report should look fine when using this branch, specifically:
- [ ] The header of the causative variant
- [ ] The columns of the dismissed variants
- [ ] Simulate a clinvar export of the variant and make sure that the clinvar form looks fine.
- [ ] On the variant page the reference allele should be also truncated and on its side there should be an icon that allows you to copy the ref allele to clipboard
- [ ] Same should be possible if the long string is the alternative allele.

Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Ready for an easy review!

**Comment by dnil** [comment]
Much better but still a few more places to go?
Screenshot 2020-03-04 at 15 28 12

**Comment by northwestwitch** [comment]
Thanks @dnil, I had missed the long names in there! I've introduced a fix that makes the text go to a new line when it's too long now, so everything shows in the report, but still looks fine:

image

**Comment by northwestwitch** [comment]
@dnil if this is fine now we could merge!

File: scout/server/blueprints/variant/templates/variant/variant.html [hunk 1]
```diff
@@ -265,7 +265,23 @@ <h6>No matching causative variants</h6>
             </td>
             <td>
               Change
-              <span><strong>{{ variant.reference }} → {{ variant.alternative }}</strong></span>
+              <span>
+                {%- if variant.reference|length > 8 -%}
+                  <strong>{{ variant.reference[:1] }}..{{variant.reference[-1:]}}</strong><button type="button" class="fa fa-file-text btn-xs js-tooltip js-copy"
+                    style="background-color: Transparent;outline:none; border: none;" data-toggle="tooltip" data-placement="bottom" data-copy="{{variant.reference}}" title="Copy to clipboard">
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/variant.html]: :+1:

File: scout/server/blueprints/variant/templates/variant/variant.html [hunk 2]
```diff
@@ -706,5 +722,37 @@ <h4 class="modal-title">Cancel Sanger sequencing</h4>
...
+    var elOriginalText = el.attr('data-original-title');
+
+    if (copyTest === true) {
+      var copyTextArea = document.createElement("textarea");
+      copyTextArea.value = text;
+      document.body.appendChild(copyTextArea);
+      copyTextArea.select();
+      try {
+        var successful = document.execCommand('copy');
+        var msg = successful ? 'Copied!' : 'Whoops, not copied!';
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/variant.html]: 😆

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/variant.html]: Yes, it's a bit funny :)

**Pull request merged**